internal/store/subscriber.go

```diff
@@ -185,10 +191,30 @@ func (e *subscriberEntry) notify(ctx context.Context, store *Store) {
 
 	activeChange := e.movePendingToActive()
 	err := e.subscriber.OnChange(ctx, store, *activeChange)
-	if err != nil {
-		store.Dispatch(NewErrorAction(err))
+	if err == nil {
+		e.clearActive()
+		return
+	}
+
+	// Backoff on error
+	backoff := activeChange.LastBackoff * 2
+	if backoff == 0 {
+		backoff = time.Second
+	} else if backoff > MaxBackoff {
+		backoff = MaxBackoff
+	}
+	logger.Get(ctx).Errorf("Problem processing change. Backing off %v. Error: %v", backoff, err)
```

Comment:
Should we include the subscriber type here? Also, what do you think about `Debug` if `backoff == 0` (i.e. first time) and then error `Error` after that?

Reply:
ooh, i like the Debug idea! done.

i left a TODO for the subscriber type - i think that's a good idea too, but i don't think we necessarily have reliable subscriber type names right now (they'll all just be "Controller")